### ISSUE TITLE
Can't install the app on 

### ISSUE THREAD

**Issue by igotabeme** [ThePacielloGroup/CCAe#141]
**Describe the bug**
The app crashes nearly 100% of the time when using the color selector. 

I deleted the app and attempted a reinstall. 

With the .zip file I receive the error, "The application “Colour Contrast Analyser (CCA)” can’t be opened.". 

With the .dmg file I receive the message: 
“Colour Contrast Analyser (CCA)” can’t be opened because Apple cannot check it for malicious software. This software needs to be updated. Contact the developer for more information. Chrome downloaded this file today at 12:36 PM from github.com."

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina 10.15.1

**To Reproduce**
Try installing either the .zip or the .dmg file on the Mac with the same OS>

**Expected behavior**
I expect the software to install, open and be usable without the crashing.

**Screenshots**
I have screen shots of the error messages as you can see below.

<img width="423" alt="Screen Shot 2019-11-12 at 12 40 22 PM" src="https://user-images.githubusercontent.com/51974570/68696021-2e41bb80-054a-11ea-81a2-7c83df0e89c7.png">

<img width="419" alt="Screen Shot 2019-11-12 at 12 39 42 PM" src="https://user-images.githubusercontent.com/51974570/68696023-2e41bb80-054a-11ea-91f9-e45cc46f497d.png">

**Comment by ferllings** [comment]
Fixed in v1.2.1
